### PR TITLE
git-now 0.1.1.0

### PR DESCRIPTION
**Pull request by dunn**

### CODE REVIEW

File: Library/Formula/git-now.rb [hunk 1]
```diff
@@ -1,37 +1,35 @@
...
-# by OSX in other shells, for example.
-__END__
---- a/git-now
-+++ b/git-now
-@@ -1,5 +1,7 @@
- #!/bin/sh
+    (bin/"git-now").write <<-EOS.undent
+      #!/bin/sh
+      PATH=#{Formula["gnu-getopt"].opt_bin}:$PATH
+      #{libexec}/bin/git-now "$@"
```
Comment (xu-cheng) [inline Library/Formula/git-now.rb]: I think in order to make it work, it should be in the same line as `PATH="#{Formula["gnu-getopt"].opt_bin}:$PATH" #{libexec}/bin/git-now "$@"`

Comment (mikemcquaid) [inline Library/Formula/git-now.rb]: Or `export` it.

Comment (dunn) [inline Library/Formula/git-now.rb]: I modeled it on [git-ftp](https://github.com/Homebrew/homebrew/blob/master/Library/Formula/git-ftp.rb#L28-30), which works as a two-liner.  The updated path remains for the duration of the subshell, doesn't it?

Comment (mikemcquaid) [inline Library/Formula/git-now.rb]: It'll find it OK it's just whether it'll be needed by subshells.

Comment (dunn) [inline Library/Formula/git-now.rb]: Oh, got it.

File: Library/Formula/git-now.rb [hunk 2]
```diff
@@ -1,37 +1,34 @@
...
 
-+alias getopt='`brew --prefix gnu-getopt`/bin/getopt'
-+
- # enable debug mode
- if [ "$DEBUG" = "yes" ]; then
-   set -x
+    zsh_completion.install "etc/_git-now"
+  end
 
+  test do
```
Comment (asparagui) [inline Library/Formula/git-now.rb]: Awesome, per usual. :+1:

**Comment by asparagui** [comment]
Thanks!

**Pull request closed without merging**